typescript/infra/scripts/core-utils.ts

```diff
@@ -0,0 +1,23 @@
+import { Contexts } from '../config/contexts';
+import { environments } from '../config/environments';
+import { DeployEnvironment } from '../src/config';
+
+import { getAgentConfig, getArgs, withContext } from './agent-utils';
+
+// utils which use both environment configs
+
+export function getEnvironmentConfig(environment: DeployEnvironment) {
+  return environments[environment];
+}
+
+export async function getConfigsBasedOnArgs(argv?: {
```

Comment:
why move this and have the diff be so big?

Reply:
see above